Magento_Checkout/web/template/minicart/item/default.html

```diff
@@ -34,10 +34,12 @@ <h3 class="minicart-product__name">
                             "
                             aria-expanded="false"
                             aria-controls="see-details"
-                            data-dropdown="see-details"
                         >
                             <!-- ko i18n: 'See Details' --><!-- /ko -->
-                            <svg class="icon dropdown-list__icon">
+                            <svg class="icon dropdown-list__icon" role="img">
+                                <title>
+                                    <!-- ko i18n: 'See Details' --><!-- /ko -->
+                                </title>
```

Comment:
Unfortunately we have to keep title tag in one line, b/c of hover effect on element otherwise it looks very strange.